hack/cluster-deploy.sh

```diff
@@ -119,7 +117,7 @@ fi
 
 if [[ "$KUBEVIRT_PROVIDER" =~ kind.* ]]; then
     #removing it since it's crashing with dind because loopback devices are shared with the host
-    _kubectl delete -n kubevirt ds disks-images-provider
+    _kubectl delete -n kubevirt ds disks-images-provider --ignore-not-found
```

Comment:
Wouldn't this be unnecessary in case of `KUBEVIRT_DEPLOY_CDI=false` as we are already removing the manifest here?
https://github.com/kubevirt/kubevirt/pull/6339/files#diff-be559fd8e5c86cd4bfb53d341659b318f4c5963a8da53ca413c19528590edd4bR38

Reply:
we have several kind types
```
kind-1.19-sriov
kind-1.19-vgpu
kind-k8s-1.17
```
only for the sriov we are setting KUBEVIRT_DEPLOY_CDI=false
so its a global guard for kind type, as the condition here is kind and not sriov or such
( we can set it for all kind types here if needed
https://github.com/kubevirt/kubevirt/pull/6339/files#diff-ce25e369a9df9edc525425cc144fc8c40599fff3cc1e5058b64336d71af67162R74 )